src/components/SchoolPicker/SchoolPicker.tsx

```diff
@@ -46,17 +46,19 @@ const SchoolPicker: FC<SchoolPickerProps> = (props) => {
       onSelectionChange={(value) => props.setSelectedSchool(value)}
       required={props.required}
     >
-      {(item) => (
-        <Item
-          key={`${item.urn}-${item.name}`}
-          textValue={`${item.name}, ${item.la}, ${item.postcode}`}
-        >
-          {formatSchoolName(
-            `${item.name}, ${item.la}, ${item.postcode}`,
-            props.schoolPickerInputValue,
-          )}
-        </Item>
-      )}
+      {(item) => {
+        const comboItemKey = HOMESCHOOL_URN
+          ? item.urn
+          : `${item.urn}-${item.name}`;
+        const comboItem =
+          item.urn === HOMESCHOOL_URN
+            ? item.name
+            : formatSchoolName(
+                `${item.name}, ${item.la}, ${item.postcode}`,
+                props.schoolPickerInputValue,
+              );
+        return <Item key={comboItemKey}>{comboItem}</Item>;
```

Comment:
i think if youre going to have a function rendered inside the item tag you need to include the "testValue" prop